converters/fromZigbee.js

```diff
@@ -5550,27 +5511,6 @@ const converters = {
             return {contact: msg.data['onOff'] === 0};
         },
     },
-    xiaomi_contact_interval: {
-        cluster: 'genBasic',
-        type: ['attributeReport', 'readResponse'],
-        options: [exposes.options.precision('temperature'), exposes.options.calibration('temperature')],
-        convert: (model, msg, publish, options, meta) => {
-            if (msg.data.hasOwnProperty('65281')) {
-                const result = {};
-                if (msg.data['65281'].hasOwnProperty('100')) {
-                    result.contact = msg.data['65281']['100'] === 0;
-                }
-
-                if (msg.data['65281'].hasOwnProperty('3')) {
```

Comment:
Same as https://github.com/Koenkk/zigbee-herdsman-converters/pull/3994/files#r826181423

Reply:
I suppose that is the same than above that you are talking about. The condition is precisely the opposite, it will publish the temperature.